src/libs/config.js

```diff
@@ -21,11 +21,27 @@ export const Config = {
 
   getGAId: async () => (await getConfig()).gaId,
 
+  getErrorApiKey: async () => (await getConfig()).errorApiKey,
+
+  getHash: async () => (await getConfig()).hash,
+
+  getTag: async () => (await getConfig()).tag,
+
   getFeatureFlag: async (featureName) => {
     const feature = _.get(await getConfig(), 'features', {});
     return _.get(feature, featureName, false);
   },
 
+  getProject: async () => {
+    const env = await Config.getEnv();
+    switch (env) {
+      case 'prod':
+        return 'broad-duos-prod';
+      default:
+        return 'broad-duos-dev';
```

Comment:
Is there no need to distinguish between staging and dev?

Reply:
Staging and dev clusters are both in the same `broad-duos-dev` google project. This will eventually change when we host the UI on the terra clusters, but until that happens, both are still in the dev one.